.travis.yml

```diff
@@ -45,7 +50,7 @@ before_script:
   - sudo pip install -r Resources/doc/requirements.txt
 
 script:
-  - phpunit ${PHPUNIT_FLAGS}
+  - SYMFONY_DEPRECATIONS_HELPER=weak phpunit ${PHPUNIT_FLAGS}
```

Comment:
And [again](https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/issues/176#issuecomment-65728870) we were getting failed builds because of deprecation warnings caused by internal Symfony code. @stof Is this the recommended way to solve this problem?